tests/python/unittest/test_numpy_op_boolean_indexing.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: skip-file
+from __future__ import absolute_import
+import sys
+import unittest
+import itertools
+import numpy as _np
+import platform
+import mxnet as mx
+import scipy.stats as ss
+import scipy.special as scipy_special
+from nose.tools import assert_raises
+from mxnet import np, npx
+from mxnet.gluon import HybridBlock
+from mxnet.base import MXNetError
+from mxnet.test_utils import same, assert_almost_equal, rand_shape_nd, rand_ndarray
+from mxnet.test_utils import check_numeric_gradient, use_np, collapse_sum_like
+from common import assertRaises, with_seed
+import random
+from mxnet.test_utils import verify_generator, gen_buckets_probs_with_ppf
+from mxnet.numpy_op_signature import _get_builtin_op
+from mxnet.test_utils import is_op_runnable, has_tvm_ops
+from mxnet.operator import get_all_registered_operators
+
+
+
+@with_seed()
+@use_np
+def test_npi_boolean_indexing():
```

Comment:
Put this under `test_np_ndarray_boolean_indexing` in test_numpy_ndarray.py. You can name this `test_boolean_indexing_assign`.